openapi/openapi.yaml

```diff
@@ -1,7 +1,8 @@
 ---
 openapi: 3.0.3
 info:
-  version: 1.0.0
+  version:
+    "$ref": "./avalanchemq_version.yaml#/version"
```

Comment:
Why not use `"$ref": "./shards.yml#/version"`? then you don't need `create_avalanchemq_version_yaml` et al.